addons/mail/static/src/core/common/thread_model.js

```diff
@@ -330,7 +330,7 @@ export class Thread extends Record {
      * @param {import("models").Persona} persona
      */
     getPersonaName(persona) {
-        return persona.displayName;
+        return persona.displayName || persona.name;
```

Comment:
`getPersonaName` should become `getPartnerName` as I don't think we ever patch it for guest, this might be a better preliminary change.

We should never need to do `|| name` here.

Reply:
instead of persona use `partner?.persona or guest.name` in the caller code?